src/region.rs

```diff
@@ -118,8 +124,9 @@ mod tests {
         assert_eq!("ca-central-1".parse(), Ok(Region::CaCentral1));
         assert_eq!("eu-central-1".parse(), Ok(Region::EuCentral1));
         assert_eq!("eu-west-1".parse(), Ok(Region::EuWest1));
+        assert_eq!("eu-west-2".parse(), Ok(Region::EuWest2));
         assert_eq!("sa-east-1".parse(), Ok(Region::SaEast1));
-        assert_eq!("us-east-1".parse(), Ok(Region::UsEast1));
```

Comment:
Looks like the old `assert_eq!` for us-east-1 was replaced with us-east-2 here.  We want both.